src/leap/mail/imap/service/imap.py

```diff
@@ -146,16 +208,8 @@ def run_service(store, **kwargs):
     :returns: the port as returned by the reactor when starts listening, and
               the factory for the protocol.
     """
-    leap_check(store, "store cannot be None")
-    # XXX this can also be a ProxiedObject, FIXME
-    # leap_assert_type(store, Soledad)
-
     port = kwargs.get('port', IMAP_PORT)
```

Comment:
I would remove the `**kwargs` from the params and add `port = IMAP_PORT` to make it explicit.